bindings/rust/aws-lc-sys-template/build.rs

```diff
@@ -217,6 +217,15 @@ fn prepare_cmake_build(build_prefix: Option<&str>) -> cmake::Config {
 
     let mut cmake_cfg = get_cmake_config();
 
+    let opt_level = env::var("OPT_LEVEL").unwrap_or_else(|_| "0".to_string());
+    if opt_level.ne("0") {
+        if opt_level.eq("1") || opt_level.eq("2") {
+            cmake_cfg.define("CMAKE_BUILD_TYPE", "relwithdebinfo");
+        } else {
+            cmake_cfg.define("CMAKE_BUILD_TYPE", "release");
+        }
+    }
```

Comment:
- What if `OPT_LEVEL` is not visible?
- Is the mapping from `1/2` to `relwithdebinfo` defined by us or defined by some docs?

Reply:
* Cargo documents the environment variable here: https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts
* The possible values for opt-level are documented here: https://doc.rust-lang.org/cargo/reference/profiles.html#opt-level
    * Its values are 0, 1, 2, 3, "s", or "z".